features/0035-report-problem/README.md

```diff
@@ -237,6 +239,12 @@ Each item in the list must be a tagged pair (a JSON {key:value}, where the key n
 
 The following is a categorization of a number of examples of errors and (current) Best Practice handling for those types of errors. The new `problem-report` message type is used for some of these categories, but not all.
 
+### Unknown Error
+
+Errors of a known error code will be processed according to the understanding of what the code means. Support of a protocol includes support and proper processing of the error codes detailed within that protocol.
+
+Any unknown error code that starts a `w.` in the DIDComm v2 style may be considered a warning, and the flow of the active protocol SHOULD continue. All other unknown error codes SHOULD be considered to be an end to the active protocol.
```

Comment:
typo, should be.... "starts with 'w.' in the"....

Reply:
good catch.